File: app/src/main/java/eu/qm/fiszki/tutorial/page/FirstPage.java
package eu.qm.fiszki.tutorial.page;

/**
 * Created by bgood on 2016-04-14.
 */
import com.cleveroad.slidingtutorial.PageFragment;
import com.cleveroad.slidingtutorial.TransformItem;

import eu.qm.fiszki.R;

public class FirstPage extends PageFragment {
    @Override
    protected int getLayoutResId() {
        return R.layout.tutorial_page_first;
    }

    @Override
    protected TransformItem[] provideTransformItems() {
        return new TransformItem[]{
                new TransformItem(R.id.TutFirstImage, true, 20),
        };
    }

}

File: app/src/main/java/eu/qm/fiszki/tutorial/page/FirstPage.kt
package eu.qm.fiszki.tutorial.page

import com.cleveroad.slidingtutorial.Direction
import com.cleveroad.slidingtutorial.PageSupportFragment
import com.cleveroad.slidingtutorial.TransformItem
import eu.qm.fiszki.R

class FirstPage : PageSupportFragment() {

    override fun getLayoutResId(): Int = R.layout.tutorial_page_first

    override fun getTransformItems(): Array<TransformItem> = arrayOf(
        TransformItem.create(R.id.TutFirstImage, Direction.LEFT_TO_RIGHT, 20f)
    )
}
